corehq/apps/user_importer/importer.py

```diff
@@ -285,15 +285,18 @@ def _get_domain_info(domain):
         domain_info = domain_info_by_domain.get(domain)
         if domain_info:
             return domain_info
-        group_memoizer = GroupMemoizer(domain)
-        group_memoizer.load_all()
+        if domain == upload_domain:
+            domain_group_memoizer = group_memoizer or GroupMemoizer(domain)
+        else:
+            domain_group_memoizer = GroupMemoizer(domain)
+        domain_group_memoizer.load_all()
```

Comment:
I looked at this again and am now wondering why `group_memoizer` is ever passed in. In the download, [this comment](https://github.com/dimagi/commcare-hq/blob/b0538659ce9295961d6516d6213a06dd15d16608/corehq/apps/users/bulk_download.py#L218-L219) implies there are performance benefits to avoiding `load_all`. But in the import, looking on master, [the call](https://github.com/dimagi/commcare-hq/blob/b0538659ce9295961d6516d6213a06dd15d16608/corehq/apps/user_importer/tasks.py#L28-L34) to this function uses the `group_memoizer` from `create_or_update_groups`, which does the [exact same thing](https://github.com/dimagi/commcare-hq/blob/b0538659ce9295961d6516d6213a06dd15d16608/corehq/apps/user_importer/importer.py#L216-L217) you're doing here. So if that's all true, now I think you could maybe just delete the param. Thoughts?

Reply:
Offlined, this is fine.